pkgs/applications/window-managers/labwc/default.nix

```diff
@@ -25,11 +27,11 @@ stdenv.mkDerivation rec {
   src = fetchFromGitHub {
     owner = "johanmalm";
     repo = pname;
-    rev = "4a8fcf5c6d0b730b1e2e17e544ce7d7d3c72cd13";
-    sha256 = "g1ba8dchUN393eis0VAu1bIjQfthDGLaSijSavz4lfU=";
+    rev = "fddeb74527e5b860d9c1a91a237d390041c758b6";
```

Comment:
Don't you need to bump `version` from `unstable-2021-02-06` to `unstable-2021-03-15` as well?
Alternatively, consider using `0.1.0` tag then applying https://github.com/johanmalm/labwc/pull/26.